ZIGSIMPlus/Views/CommandOutputViewController.swift

```diff
@@ -6,18 +6,18 @@
 //  Copyright © 2019 1→10, Inc. All rights reserved.
 //
 
-import UIKit
 import MediaPlayer
+import UIKit
 
 class CommandOutputViewController: UIViewController {
-    @IBOutlet weak var textField: UITextView!
-    @IBOutlet weak var touchArea: UIView!
-    @IBOutlet weak var textPreview: UIView!
-    @IBOutlet weak var imagePreview: UIView!
-    @IBOutlet weak var togglePreviewModeButton: UIBarButtonItem!
+    @IBOutlet var textField: UITextView!
+    @IBOutlet var touchArea: UIView!
+    @IBOutlet var textPreview: UIView!
+    @IBOutlet var imagePreview: UIView!
+    @IBOutlet var togglePreviewModeButton: UIBarButtonItem!
```

Comment:
IBOutlets should be weak except for special cases, shouldn't they?
https://qiita.com/chocovayashi/items/a96adc1356b7c45524b7

Reply:
Apple has changed the opinion about holding views; now they recommend strong refs except for refs to views back up the view hierarchy.
https://stackoverflow.com/a/31395938

Reply:
Oh I didn't know that.
Thanks!